### DISCUSSION TITLE
ryzen error

### DISCUSSION THREAD

**Discussion by MattDA1** [jef/streetmerchant#963] [Help]
so, every time I get a stock notification for the AMD ryzen 5900x it's the same thing
the screenshot that records it shows me "stock will be available on January 16th, 2021" but the link that I'm getting says "

These items are currently unavailable:
--
AMD Ryzen 9 5900X 12-core, 24-Thread Unlocked Desktop Processor Without Cooler"

is there a solution? anyone have any idea?
my screenshot:
![image](https://user-images.githubusercontent.com/74619697/100393696-38463780-3043-11eb-8cfe-36fcfd40a7fb.png)

automatically recorded screenshot:
![success-1606427691296](https://user-images.githubusercontent.com/74619697/100393702-3da38200-3043-11eb-81f9-2b16034e5515.png)

**Comment by variable42** [comment]
Amazon and Newegg don't prevent bots from auto-purchasing. Other bots are automatically scooping up the inventory before you get a chance to do it manually.

**Comment by achal1012** [comment]
I am seeing the same behavior as @MattDA1.
